README.md

```diff
@@ -104,6 +104,15 @@ steps:
     NUGET_AUTH_TOKEN: ${{secrets.AZURE_DEVOPS_PAT}} # Note, create a secret with this name in Settings
 - name: Publish the package to Azure Artifacts
   run: dotnet nuget push <my project>/bin/Release/*.nupkg
+
+# Authticates packages to push to NuGet Org
```

Comment:
Also, there is probably a typo in all these comments: `Authticates` -> `Authenticates`